### PR TITLE
NOJIRA: fix smoke test adding retry

### PR DESCRIPTION
**Pull request by samuelzerozero**
*fix smoke test retry

### CODE REVIEW

File: e2e/tests/smoke_test.js [hunk 1]
```diff
@@ -8,6 +8,6 @@ Scenario('Sign in as local authority and create a case', async ({I, caseListPage
   const caseName = `smoke test case (${new Date().toISOString()})`;
   const caseId = await I.logInAndCreateCase(config.swanseaLocalAuthorityUserOne, caseName);
   await I.navigateToCaseList();
-  await caseListPage.searchForCasesWithName(caseName, 'Open');
+  await I.retryUntilExists(() => caseListPage.searchForCasesWithName(caseName, 'Open'), `//ccd-search-result/table//tr[//a[contains(@href,'${caseId}')]]`);
```
Comment (tomaszpowroznik) [inline e2e/tests/smoke_test.js:11]: ideally retries should be inside caseListPage.searchForCasesWithName and selector should be in page object

**Review by tomaszpowroznik — APPROVED** [review]

File: e2e/tests/smoke_test.js (hunk 1 above)
Comment (lewisbirks) [inline e2e/tests/smoke_test.js:11]: I was thinking of instead changing the timeouts for the search stuff to be available

Comment (lewisbirks) [reply e2e/tests/smoke_test.js:11]: my current changes have increased timeout on several waitForElements and it worked 4/5 times with the only failure being the `net::ERR_ABORTED` error

Comment (tomaszpowroznik) [reply e2e/tests/smoke_test.js:11]: the problem is different, timeout wont help. It is not that it loads slowly.  The issue is that case is created, and not yet propagated from db->ES, and we try to search for it in ES. Thats why test needs to click search multiple time and repeat request until case is in ES

**Pull request merged**